### DISCUSSION TITLE
[6 settembre 2023] Calcolabilità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#17] [Calcolabilità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/fa1d7ef1-d7b8-4251-8993-2d7f5db16a0c)

**Comment by Exyss** [comment]
Il primo punto è un modo alternativo per chiedere di dimostrare l'esercizio #14 (difatti risulta facilmente evidente che la macchina $M'$ costruita dalla la riduzione sia un riconoscitore che potrebbe andare in loop)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by daromacarlo** [comment]
_**Punto 1)**_
<img width="1569" height="566" alt="immagine" src="https://github.com/user-attachments/assets/92cb4ade-37cf-4012-9e88-80f26fdbc57b" />
<img width="1568" height="260" alt="immagine" src="https://github.com/user-attachments/assets/62579c13-e68e-4f13-a063-d42070e8c4ca" />
